pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -558,6 +576,16 @@ func (ssr *Snapshotter) snapshotEventHandler(stopCh <-chan struct{}) error {
 			if err != nil {
 				return err
 			}
+			if ssr.healthConfig.Enabled {
+				ctx, cancel := context.WithTimeout(context.TODO(), ssr.config.DeltaSnapshotPeriod.Duration)
+				if err = heartbeat.UpdateFullSnapshotLease(ctx, ssr.logger, ssr.PrevFullSnapshot, ssr.k8sClientset, ssr.healthConfig.FullSnapshotLeaseName); err != nil {
+					ssr.logger.Warnf("1 Failed to update full snapshot lease with error: %v", err)
```

Comment:
What does `1` mean?

Reply:
It was used for local debugging. Meant to be removed, however missed it last time. 
[here](https://github.com/gardener/etcd-backup-restore/pull/382#discussion_r719575321)

Reply:
Thanks